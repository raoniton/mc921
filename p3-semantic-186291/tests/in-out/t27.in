class Main {
    public static void main(String[] args) {
        n=n+1;
        n=n-1;
        n=n%0;
        n=n*10;
        n=n/4;
    }
}