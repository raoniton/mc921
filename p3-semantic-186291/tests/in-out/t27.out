SemanticError: n is not defined @ 3:11
